### PR TITLE
Fixing misspelling

### PR DESCRIPTION
**Pull request by feargswalsh92**
I fixed a misspelling of  MultivaluedOnlyReorderController I found on line 1533 of ViewController.swift, this is my first ever pull request so apologies in advance if I didn't do this properly but I read the Contributing guidelines about fifty times so I think I'm good.

### CODE REVIEW

File: Example/Example/ViewController.swift [hunk 1]
```diff
@@ -1530,7 +1530,7 @@ class MultivaluedController: FormViewController {
     }
 }
 
-class MultivaluedOnlyRearderController: FormViewController {
```
Comment (m-revetria) [inline Example/Example/ViewController.swift]: This change must be reflected in the Storyboard where this class is used.

File: Eureka.xcworkspace/contents.xcworkspacedata [hunk 2]
```diff
@@ -10,4 +10,7 @@
    <FileRef
       location = "group:Example.xcodeproj">
    </FileRef>
+   <FileRef
```
Comment (m-revetria) [inline Eureka.xcworkspace/contents.xcworkspacedata]: Please revert these changes as they are not related with the proposed change.

File: Example/Example/ViewController.swift (hunk 1 above)
Comment (mtnbarreto) [inline Example/Example/ViewController.swift]: Will be fixed in Eureka Xcode 9 compatible version.

**Comment by mtnbarreto** [comment]
@feargswalsh92 Thanks for reporting the issue. We will fix it in Xcode 9 compatible version that will be released shortly.
